### DISCUSSION TITLE
When will be Docker images published to Docker Hub?

### DISCUSSION THREAD

**Discussion by ottobolyos** [mtconnect/cppagent#339] [Q&A] [answered]
@wsobel, I’d like to inform where will be Docker images published to Docker Hub?

And what is the `DOCKERHUB_USERNAME` in [`.github/workflows/build-docker-image.yml` (L71)](https://github.com/mtconnect/cppagent/blob/v2.2.0.8/.github/workflows/build-docker-image.yml#L71)? `mtconnect`, just like it is suggested in [`.github/workflows/build-docker-image.yml` (L9)](https://github.com/mtconnect/cppagent/blob/v2.2.0.8/.github/workflows/build-docker-image.yml#L9).

**Comment by wsobel** [comment, marked as answer]
The docker images are published under `mtconnect`. See https://hub.docker.com/u/mtconnect

There are two images: `mtconnect/agent` and `mtconnect/demo`. The demo runs is the same content as https://demo.mtconnect.org/  and 
the agent can be configured as described in https://github.com/mtconnect/cppagent/tree/main/docker.

You can also check out the `demo/compose` directory for some ideas.

The DOCKERHUB_USERNAME is mtconnect

Best,
- W
